lua/telescope-orgmode/typehints.lua

```diff
@@ -0,0 +1,61 @@
+-- Type-hints copied from nvim-orgmode to simplify development
```

Comment:
Why these need to be copied over?

Reply:
The LSP doesn't see types from a different lua module during development. Copying them into this file helped me a lot to see the types which got returned from the API functions.

Is there a way to tell the LSP where to look for them without copying them over?

Reply:
Are you using https://github.com/folke/neodev.nvim ?

If yes, you can have it load all the plugins in the LSP so you get the autocompletion like this:

```lua
  require('neodev').setup({
    override = function(_, options)
      if not options.plugins then
        options.plugins = true
      end
      return options
    end,
  })
```

This is happening anyway when you are loading your config, but when you develop in a plugin it needs to explicit. I use only this by default since I don't need other:

```lua
  require('neodev').setup({
    override = function(_, options)
      if not options.plugins then
        options.plugins = { 'nvim-treesitter', 'plenary.nvim' }
      end
      return options
    end,
  })
```

Reply:
Thanks a lot. I stumbled upon this plugin a while ago but didn't take a deeper look. Will catch up on this.

Reply:
For now I will keep the typehints here in the Telescope plugin. I actually installed the plugin but I'm probably to dumb and too impacient to configure it correctly for plugin development with nvim-orgmode. 

Would you mind to provide a correct setup as a hint in the contributor section of the nvim-orgmode documentation?